tests/spread/integration/iptables/task.yaml

```
@@ -0,0 +1,23 @@
+summary: Integration tests for iptables
+
+execute: |
+  rootfs="$(install-slices passwd_config libpam-runtime_config netbase_default-hosts sudo_bins iptables_bins)"
+  host=$(hostname)
+  
+  # setup /etc/hosts for the current hostname, otherwise
+  # sudo will be confused
+  sed -i "/127.0.0.1\tlocalhost/a 127.0.0.1\t$host" "$rootfs"/etc/hosts
```

```suggestion
  sed -i "/127.0.0.1\slocalhost/a 127.0.0.1\t$host" "$rootfs"/etc/hosts
```
On my 24.04 system they are space-separated, so better use `\s` to make it work across all systems.